### PR TITLE
Fix crash for UI modification on a background thread.

### PR DESCRIPTION
**Pull request by roberthein**
Fix crash for UI modification on a background thread.

### CODE REVIEW

File: Toshi/Controllers/Messaging/ChatInteractor.swift [hunk 1]
```diff
@@ -142,17 +142,22 @@ final class ChatsInteractor: NSObject {
...
                     guard let txHash = json["tx_hash"] as? String else { fatalError("Error recovering transaction hash.") }
                     guard let value = parameters["value"] as? String else { return }
 
                     let payment = SofaPayment(txHash: txHash, valueHex: value)
-                    self?.sendMessage(sofaWrapper: payment)
-
-                    completion?(true)
+                    
+                    DispatchQueue.main.async {
+                        self?.sendMessage(sofaWrapper: payment)
```
Comment (yuliaveres) [inline Toshi/Controllers/Messaging/ChatInteractor.swift]: maybe better to send in background ?

Comment (roberthein) [reply Toshi/Controllers/Messaging/ChatInteractor.swift]: What do you mean exactly?

Comment (roberthein) [reply Toshi/Controllers/Messaging/ChatInteractor.swift]: Yes! Ok 👌

Comment (roberthein) [reply Toshi/Controllers/Messaging/ChatInteractor.swift]: done!

**Comment by yuliaveres** [comment]
Nice catch !

**Pull request merged**